<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>specific</class>
 <widget class="QDialog" name="specific">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene Visualization</string>
  </property>
  <widget class="QTabWidget" name="tabPage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>781</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_smenu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cost">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>470</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cost_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>470</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_items">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ITEMS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_items">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>181</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_tree">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 170, 142);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Trees</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_flowers">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 170, 142);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Flowers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ground">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 170, 142);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Ground Covers</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_flower">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_flowerback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_flower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 185, 160);
border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="region">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Region:</string>
    </property>
   </widget>
   <widget class="QLabel" name="f_spacing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spacing:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblf_water">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watering:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblf_price">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_region">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_spacing">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_water">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_tree">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>201</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_treeback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="region_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Region:</string>
    </property>
   </widget>
   <widget class="QLabel" name="f_spacing_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spacing:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblf_water_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watering:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblf_price_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_region_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_spacing_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_water_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 185, 160);
border:none;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_ground">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>110</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_groundback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="region_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Region:</string>
    </property>
   </widget>
   <widget class="QLabel" name="f_spacing_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spacing:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblf_water_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watering:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblf_price_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_region_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_spacing_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_water_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_price_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 185, 160);
border:none;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_uploadbg">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 164, 79);
border: none;
color: white;</string>
   </property>
   <property name="text">
    <string>Upload Background</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radiobtn_beach">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radiobtn_hotel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radiobtn_default">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cost_3">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>470</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Baht</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbl_bg</zorder>
  <zorder>tabPage</zorder>
  <zorder>btn_smenu</zorder>
  <zorder>lbl_cost</zorder>
  <zorder>lbl_cost_2</zorder>
  <zorder>lbl_items</zorder>
  <zorder>frame_items</zorder>
  <zorder>frame_flower</zorder>
  <zorder>frame_tree</zorder>
  <zorder>frame_ground</zorder>
  <zorder>btn_uploadbg</zorder>
  <zorder>radiobtn_beach</zorder>
  <zorder>radiobtn_hotel</zorder>
  <zorder>radiobtn_default</zorder>
  <zorder>lbl_cost_3</zorder>
 </widget>
 <resources>
  <include location="bg.qrc"/>
 </resources>
 <connections/>
</ui>
